common/lib/models/comment.dart

```diff
@@ -48,9 +52,18 @@ class Comment extends CommentBase {
 
   factory Comment.fromJson(Json json) => _$CommentFromJson(json);
 
+  Map<String,dynamic> get details =>
```

Comment:
What is this used for?

Reply:
Same as for common/lib/models/poll.dart `Map<String,dynamic> get details`